src/auth/.gcb/integration.yaml

```diff
@@ -16,12 +16,12 @@ options:
   dynamic_substitutions: true
   substitutionOption: 'ALLOW_LOOSE'
   logging: CLOUD_LOGGING_ONLY
+  env:
+    - GOOGLE_CLOUD_PROJECT=${PROJECT_ID}
```

Comment:
Consider adding a cache, these builds can get slow .

https://github.com/googleapis/google-cloud-rust/blob/ce2b7195a0ee696cbd0eeb570b8c7ec78a822554/.gcb/integration.yaml#L23

and

https://github.com/googleapis/google-cloud-rust/blob/ce2b7195a0ee696cbd0eeb570b8c7ec78a822554/.gcb/integration.yaml#L29C1-L36C40

Reply:
My plan was to add the build cache after the build is set up.

There is a new line item in #806 to track the task.